src/components/calcite-popover/calcite-popover.tsx

```diff
@@ -20,9 +20,8 @@ import {
 import { StrictModifiers, Placement, Instance as Popper } from "@popperjs/core";
 import { VNode } from "@stencil/core/internal/stencil-core";
 import { guid } from "../../utils/guid";
-import { HOST_CSS } from "../../utils/dom";
```

Comment:
this const can be removed from the utility file.

Reply:
Did this but never saved the file 🤦